src/popup/router/components/Modals/ShareQr.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <Default
+    v-bind="{ ...$attrs, resolve }"
+    :title="$t('modals.shareQr.title')"
+    :close="resolve"
+  >
+    <div>
+      <qrcode-vue
+        :value="qrStr"
+        class="qrcode"
+        size="170"
+        level="Q"
+      />
+    </div>
+    <TemplateRenderer
+      :class="!msgStr.includes('.chain') && 'address'"
+      :str="msgStr"
+    />
+    <template slot="footer">
+      <Button
+        fill="secondary"
+        @click="resolve"
+      >
+        {{ $t('modals.shareQr.close') }}
+      </Button>
+      <Button
+        v-if="!IS_MOBILE_DEVICE"
+        v-clipboard:copy="qrStr"
+        icon-text
+      >
+        <Copy />
```

Comment:
When you click on copy the qr code should be also highlighted
![image](https://user-images.githubusercontent.com/9008074/136730451-880aff6b-943a-49f7-86b3-e40f5c852fe8.png)